newrelic/data_source_newrelic_alert_channel.go

```diff
@@ -16,22 +16,26 @@ func dataSourceNewRelicAlertChannel() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the alert channel in New Relic.",
 			},
 			"type": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Alert channel type, either: `email`, `opsgenie`, `pagerduty`, `slack`, `victorops`, or `webhook`.",
```

Comment:
Can we generate the list of accepted types programmatically as was done in another file:
```
fmt.Sprintf("(Required) The type of channel. One of: (%s).", strings.Join(validAlertChannelTypes, ", ")),
```

Reply:
👍